KILTprotocol/polkadot-apps
# chore: update to newest version of chain explorer

## closes KILTProtocol/ticket#538

- merge all changes from upstream project (newest merged commit 7cb31f6 July 30)
- add KILT dev and prod chain as an option
- add prettier config to resolve linting conflicts after auto saves

## How to test:

```
yarn install
yarn start
```

If you want to test it with the devnet, you have to add the custom types to your metadata, as the spec points to the testnet:
```json
{
  "Address": "AccountId",
  "BlockNumber": "u64",
  "DelegationNodeId": "Hash",
  "ErrorCode": "u16",
  "Index": "u64",
  "LookupSource": "AccountId",
  "Permissions": "u32",
  "PublicBoxKey": "Hash",
  "PublicSigningKey": "Hash",
  "Signature": "MultiSignature"
}

```

We probably should change the text for the chains. Also our live chain is more like a test chain and the dev chain should maybe not be listed there? Not sure about that.

## Checklist:
